tests/plugins/image2/manual/defaultlockratio.md

```diff
@@ -0,0 +1,20 @@
+@bender-tags: 4.20.0, feature, 5219
```

Comment:
Could you extend that test with 2 other editors and different values for the configuration option? It will let us test if everything is fine with the old behavior during the testing phase. I didn't notice any other dedicated manual test for the lock button, but we can just add the proper ticket number to the manual test if such a test is available (e.g. we could probably enable `aspectratio.html` test).